docs/user_manual.md

```diff
@@ -23,6 +24,17 @@ Note that this guide assumes that you are familiar with [Docker](http://docker.i
   * [Using Deployment Groups](#using-deployment-groups)
 * [Once Inside The Container](#once-inside-the-container)
 
+
+
+Source Code Setup
+---
+(For Helios Developers and Contributors)
+1) Under the parent level helios source directory run "mvn clean install." This will run the integration tests, which may take some time. To avoid running the tests, add the "-DskipTests" option. 
```

Comment:
Use backticks (`) instead of double-quotes (") for commands, filenames, etc., to get the proper Markdown formatting.